_tools/check_circular_submodule_dependencies.ts

```diff
@@ -38,7 +38,7 @@ async function check(
     }
   }
   deps.delete(submod);
-  deps.delete("types.d.ts");
+  deps.delete("io/types.d.ts");
```

Comment:
I think you should remove this line instead.

Reply:
That's correct

Reply:
Fixed